Abhilash107/Sidebar
# added resposiveness and animation

This PR adds hover animation over elements of sidebar and also makes website responsive to different screen sizes.

Fixes #53

Test:

- Go to the sidebar.
- hover over different items of sidebar and see the animated hovering.

[hover.webm](https://github.com/user-attachments/assets/bf677246-27dd-4cf2-be70-cd4d8f846541)
